cmd/config.go

```diff
@@ -206,6 +206,17 @@ func initConfig(cmd *cobra.Command) error {
 		config.cl[name] = cl
 	}
 
+	// override chain if needed
```

Comment:
This would override the chain in GetDefaultClient. Is there another reason to get the value directly from cmd there?

Reply:
Oh I didn't see this. Thats a cool way of doing it let me test it.